Application/EdFi.Ods.Api/Security/Authorization/Repositories/AggregateRootCriteriaProviderDecoratorBase.cs

```diff
@@ -56,77 +56,101 @@ public ICriteria GetCriteriaQuery(TEntity specification, IQueryParameters queryP
-                x => x.SubjectEndpointName,
-                x => x);
+                var allFiltersGroupedBySubjectName = authorizationFilters.GroupBy(
+                    x => x.SubjectEndpointName,
+                    x => x);
 
-            // ICriterions combined using AND
-            var conjunction = new Conjunction();
+                // ICriterions combined using AND
+                var conjunction = new Conjunction();
 
-            foreach (var subjectNameGrouping in allFiltersGroupedBySubjectName)
-            {
-                // ICriterions combined using OR
-                var disjunction = new Disjunction();
+                foreach (var subjectNameGrouping in allFiltersGroupedBySubjectName)
+                {
+                    // ICriterions combined using OR
+                    var disjunction = new Disjunction();
 
-                var unsupportedAuthorizationFilters = new List<string>();
+                    bool isSubjectNameAuthorizable = false;
+                    var unsupportedAuthorizationFilters = new List<string>();
 
-                bool authorizationFilterApplied = false;
-                
-                foreach (var filterDetails in subjectNameGrouping)
-                {
-                    if (!_authorizationCriteriaApplicatorProvider.TryGetCriteriaApplicator(
-                        filterDetails.FilterName,
-                        typeof(TEntity),
-                        out IReadOnlyList<Action<ICriteria, Junction, IDictionary<string, object>, JoinType>> applicators))
+                    foreach (var filterDetails in subjectNameGrouping)
                     {
-                        unsupportedAuthorizationFilters.Add(filterDetails.FilterName);
+                        IReadOnlyList<Action<ICriteria, Junction, IDictionary<string, object>, JoinType>> applicators;
 
-                        continue;
-                    }
+                        if (!_authorizationCriteriaApplicatorProvider.TryGetCriteriaApplicator(
+                            filterDetails.FilterName,
+                            typeof(TEntity),
+                            out applicators))
+                        {
+                            unsupportedAuthorizationFilters.Add(filterDetails.FilterName);
 
-                    // Invoke the filter applicators against the current query
-                    foreach (var applicator in applicators)
-                    {
-                        var parameterValues = new Dictionary<string, object>
+                            continue;
+                        }
+
+                        isSubjectNameAuthorizable = true;
+
+                        var filtersBackedByNewAuthViews = new List<string>
                         {
-                            {"SourceEducationOrganizationId", filterDetails.ClaimValues},
-                            {filterDetails.ClaimEndpointName, filterDetails.ClaimValues},
```

Comment:
I haven't checked, but I can't believe this actually would actually merge cleanly with `main`. But regardless, the `filtersBackedByNewAuthViews` is part of what I cleaned up right before the v5.3 release, right? We definitely don't want this hard-coded list coming back into the code base.

It makes me question the lineage of this PR. What branch/commit is this based on? Why is this showing up as re-inserted by the record ownership work? It looks like something is very wrong here because Arizona wouldn't have made these changes -- we introduced this variable on an older commit that is long out of date, and is not what is in `main` anymore.

Reply:
I noticed that you have reverted old changes and sync in main